infra/mock_api/template/rds.tf

```diff
@@ -1,3 +1,35 @@
+# generates a secure password randomly
+
+# when 176 is merged:
+# update: common/mock_api_db/POSTGRES_PASSWORD in actual parameter store to prevent conflicts
+# update: what the rds setup considers a password
+# update: add updated auth token to ecs task
+
+resource "random_password" "random_db_password" {
+  length           = 48
+  special          = true
+  min_special      = 6
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
+resource "aws_ssm_parameter" "random_db_password" {
+  name  = "/common/mock_api_db/POSTGRES_PASSWORD"
+  type  = "SecureString"
+  value = random_password.random_db_password.result
+}
+
+# Creates an API key we can use to auth containers with
+resource "random_password" "random_api_key" {
+  length           = 16
+  special          = true
+  min_special      = 6
+  override_special = "!#$%&*()-_=+[]{}<>:?"
```

Comment:
FYI these overrides are specifically to match the allowed characters in PSQL databases — I'm not sure if you need them for the API key.

(but it's definitely easier to keep this here!)